Gamemode Mods/Stable/Starcore_Sharetrack/Data/Scripts/ShipPoints/HudPointsList.cs

```diff
@@ -98,11 +102,14 @@ private void ShiftTCalcs(IMyCubeGrid focusedGrid)
             // Update once per second
             if (MatchTimer.I.Ticks % 60 != 0)
                 return;
-
+            var stopwatch = new System.Diagnostics.Stopwatch();
+            stopwatch.Start();
             ShipTracker shipTracker;
-            TrackingManager.I.TrackedGrids.TryGetValue(focusedGrid, out shipTracker);
-            if (shipTracker == null)
+            if (!TrackingManager.I.TrackedGrids.TryGetValue(focusedGrid, out shipTracker))
+            {
                 shipTracker = new ShipTracker(focusedGrid, false);
+                TrackingManager.I.TrackedGrids[focusedGrid] = shipTracker;
```

Comment:
This is adding the tracker to the TrackedGrids dictionary, which is for Shift-M only. Would suggest using a class-level variable or similar instead.